nextcord/http.py

```diff
@@ -151,48 +472,172 @@ def __exit__(
 
 
 class HTTPClient:
```

Comment:
Will this be in the actual documentation? If so, remember to add it to `api.rst`.

Reply:
...not at this time. Once Nextcord/Client/Bot becomes less of a single monolithic structure and more build-a-bot with separation between the various parts sure, but until then I'd like to keep it off the docs.